src/client/dfuse/dfuse_main.c

```diff
@@ -217,21 +217,64 @@ dfuse_launch_fuse(struct dfuse_projection_info *fs_handle,
 	return false;
 }
 
+static void
+show_version(char *name)
+{
+	fprintf(stdout, "%s version %s, libdaos %d.%d.%d\n",
+		name, DAOS_VERSION, DAOS_API_VERSION_MAJOR,
+		DAOS_API_VERSION_MINOR, DAOS_API_VERSION_FIX);
+	fprintf(stdout, "Using fuse %s\n", fuse_pkgversion());
+};
+
 static void
 show_help(char *name)
 {
-	printf("usage: %s -m=PATHSTR -s=RANKS\n"
+	printf("usage: %s -m mountpoint\n"
+		"Options:\n"
 		"\n"
-		"	-m --mountpoint=PATHSTR	Mount point to use\n"
-		"	   --pool=<name>	pool UUID/label\n"
-		"	   --container=<name>	container UUID/label\n"
+		"	-m --mountpoint=<path>	Mount point to use\n"
+		"\n"
+		"	   --pool=label		pool UUID/label\n"
+		"	   --container=label	container UUID/label\n"
 		"	   --sys-name=STR	DAOS system name context for servers\n"
-		"	-S --singlethreaded	Single threaded\n"
-		"	-t --thread-count=COUNT Number of fuse threads to use\n"
+		"\n"
+		"	-S --singlethread	Single threaded\n"
```

Comment:
Is it safe to change `--singlethreaded` to `--singlethread`?
One such instance is:
https://github.com/daos-stack/daos/blob/cf0db8517c62c91dbd14079cd6e7220fb3848a7b/src/tests/ftest/util/dfuse_utils.py#L28
But it doesn't appear that anything else in the DAOS code base is using `--singlethread`